### DISCUSSION TITLE
transfer-05-file-transfer-cloud- Error: missing or invalid value for key bucketname, missing or invalid value for key region

### DISCUSSION THREAD

**Discussion by nimritee** [eclipse-edc/Samples#222] [Q&A]
Hello Community,

Samples: transfer-05-file-transfer-cloud. ReadMe seems to be outdated. Need to perform several steps mentioned in [Pull Request](https://github.com/eclipse-edc/Samples/pull/128) to run the consumer and provider jar. Still there is an error while transferring files to the AWS bucket.

Filetransfer.json- ` {
  "@context": {
    "edc": "https://w3id.org/edc/v0.0.1/ns/"
  },
  "@type": "TransferRequestDto",
  "dataDestination": {
    "type": "AmazonS3",
    "region": "us-east-1",
    "bucketName": "<Unique-bucket-name>-src-bucket",
    "keyName": "keyName"
  },
  "protocol": "dataspace-protocol-http",
  "assetId": "1",
  "contractId": "d2c85968-984d-4268-9ce7-d673cf423faa",
  "connectorId": "provider",
  "connectorAddress": "http://localhost:8282/protocol",
  "transferType": {
    "contentType": "application/octet-stream",
    "isFinite": true
  },
  "privateProperties": {}
}`

Following is the error although everything is provided in filetransfer.json:
> Fatal error while Initiate data flow. Error details: Missing or invalid value for key bucketName, Missing or invalid value for key region

**Comment by yassirahla** [comment]
Hello, are there any updates on the AWS S3 file transfer issue?
